astor/code_gen.py

```diff
@@ -19,15 +19,18 @@
 
 import ast
 import sys
+import gc
 
 from .op_util import get_op_symbol, get_op_precedence, Precedence
 from .node_util import ExplicitNodeVisitor
-from .string_repr import pretty_string
-from .source_repr import pretty_source
+from .prettifier import StringLiteral, beautify
 
+###################################################################
+#               Main interface
+###################################################################
 
 def to_source(node, indent_with=' ' * 4, add_line_information=False,
-              pretty_string=pretty_string, pretty_source=pretty_source):
```

Comment:
This is an interesting case.  These weren't there in 0.5 (pretty printer added right after that), and I unthinkingly already changed the call signature to pretty_source a couple of commits ago, and the internals are now way different.

Having said that, I think we can actually do a reasonable job of supporting that interim API with the current code, so I'll go back and look at that.  I'm at (real) work today, so that will have to wait until Thursday.

Also, I think that it might be useful to make a `ToSource` class to make things more easily configurable, and make to_source a convenience method of that class.